### ISSUE TITLE
GitHub wiki is outdated

### ISSUE THREAD

**Issue by cleveHEX** [Libre-TrainSim/Libre-TrainSim#536]
### Summary

Wiki on the GitHub has some typos and dead links (pointing to the old repository). If you give me a permission to edit it, I would try to fix it.

### How to reproduce

Visit GitHub wiki of this repository.

### Observed behavior

Links are dead

### Expected behavior

Links are not dead

### Version

N/A

**Comment by HaSa1002** [comment]
Yeah , we're moving to https://github.com/Libre-TrainSim/lts-docs/pull/1 deployed to https://docs.libretrainsim.org/en/merge-pages. I started with slightly improving the docs and noticed how much is outdated and outright unhelpful or fluff

**Comment by HaSa1002** [comment]
What happens when you try to edit and save a page?

**Comment by cleveHEX** [comment]
Do you mean edit and save the page on GH wiki? I simply can't, there is no button, you can try to look at other public repos. However if desired, the wiki can be cloned locally and its files can be copied to the repo you sent.

**Comment by HaSa1002** [comment]
Interesting because I had a button on another public repo 👀

**Comment by HaSa1002** [comment]
ah nevermind. Have fun with the wiki

**Comment by HaSa1002** [comment]
There's a checkbox in the repo settings
